user_guide/common_features_channels.rst

```diff
@@ -3,13 +3,45 @@
 Channels
 ========
 
-Collection of comprehensive customer-related data is a cornerstone of customer relations management.
 
-To represent different sources of customer-related data, OroCRM uses records of the Channel entity ("channels").
+Today, successful businesses usually have more than one sales site. These may be different online stores, 
+business-to-business enterprises, or business-to-customer outlets. Moreover, different customer and sales-related 
+information can be received from various survey-campaigns or membership in clubs, funds, charity events, etc. Gaining a 
+full understanding of all the information you receive from each of these sources, along with all the individual customer 
+information you collect from various other sources, becomes crucial. With OroCRM's Channels you can do both with minimum 
+effort. 
 
-For example, any store, on-line site, social activity, etc. may be represented with its own channel.  
+In OroCRM, each Channel record represents one sources of customer-related data.
+Upon creation, you can choose what kind of customer-related information will be collected from each source (in technical 
+words "what entities will be assigned to this channel"). 
+
+For example, you have two channels of customer related data: your on-line store and your office.
+While they are both related to your customers, the information and customer relations process is quite different.
+ 
+Of course, there are much more details and peculiarities, but if we keep it simple, in he on-line store, somebody comes 
+to the store's website, looks through the items offered, and if interested in 
+buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the 
+web-store customers can log-in and their profile details will be used. What kind of information do you need to make 
+relations with such customers successful? The most basic things are - what pages have people viewed, what items they 
+have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some 
+complaints after the delivery.
+
+It is quite different when you are to execute an agreement in your office, there may be some complex arrangements, 
+details whereof you want to track, some preliminary agreements and executed deeds. You can have several ongoing and 
+perspective projects with one big company, but with different project managers, so you need their contact details, as 
+well as understanding of their position in the company and relation to the project. 
```

Comment:
Edit: In contrast, it is quite different when a sale takes place between two businesses. For instance, there may be a number of complex arrangements you would want to keep track of, such as any preliminary agreements or executed deeds. You may also have several ongoing projects with a single company, all of which involve multiple project managers you will need contact details for. You will also need to understand their position in the company and their relation to the project.